Parse/TPTP.hpp

```diff
@@ -852,6 +860,18 @@ class TPTP
   /** This field stores names of input units if the
    * output_axiom_names option is enabled */
   static DHMap<unsigned, vstring> _axiomNames;
+
+  /**
+   * During question parsing, we store the mapping from int variables
```

Comment:
Do we expect this to be used only for TPTP and not for SMT-LIB inputs?

Reply:
I was only coding this up with the TPTP users in mind. Also, not that this only kicks in (for performance reasons) only for the tptp role "question". 

Not sure what the analogue would be in SMT-LIB, but I am definitely not against it.

Reply:
For synthesis, we add the answer literals for any conjecture in the form `(assert-not ([forall (...) (] exists ... [)] ))`.
The `assert-not` is somewhat non-standard (in addition to us also Z3 supports it, but not cvc5 for example), but we've been using it for conjectures for some time already.